docs/dev/recommended-extension.md

```diff
@@ -0,0 +1,9 @@
+# Recommended Extensions
```

Comment:
minor nit: should the filename and heading match? i.e. should the filename be `recommended-extensions`?